Applications/shapeworks/MeshCommands.cpp

```diff
@@ -1179,6 +1220,7 @@ void CompareMesh::buildParser()
   parser.prog(prog).description(desc);
 
   parser.add_option("--name").action("store").type("string").set_default("").help("Compare this mesh with another.");
+  parser.add_option("--epsilon").action("store").type("double").set_default(0.0).help("Epsilon [default: %default].");
```

Comment:
One suggestion I made earlier was to default this to -1 and thereby not use it at all unless explicitly specified. Our existing tests have all passed fine without epsilon, so why start using it for everything now? Consider comparing massive values that might even differ in the 10s compared to tiny values that might not differ more than 10e-10. Epsilon default are not easy to generalize, which is why significant digits are a (somewhat) better thing to compare.

Reply:
Done